server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java

```diff
@@ -194,7 +195,14 @@ private void initSqlIterator() {
         }
 
         for (String sqlObjectId : parameters.getSqlObjectIds()) {
-            try (InputStream current = cloudObjectStorageService.getObject(sqlObjectId)) {
+            InputStream inputStream;
```

Comment:
when to close this stream, may add comments

Reply:
Have been resolved.